pallets/inflation-manager/src/migrations.rs

```diff
@@ -25,7 +25,12 @@ mod upgrade {
 			let current = Pallet::<T>::current_storage_version();
 
 			if onchain_storage_version < current {
-				let do_initialize_at = T::DoInitializeAt::get();
+				InitialBlock::<T>::put(T::DoInitializeAt::get());
```

Comment:
We can optimise reads if we put DoInitializeAt in a variable at first, then put that into storage where needed going forward and use it in logic also.